<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceApp</class>
 <widget class="QWidget" name="ReferenceApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>187</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>487</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reference App</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resurces/ReferenceApp.qrc">
    <normaloff>:/logo/AT_SERVER_alpha.png</normaloff>:/logo/AT_SERVER_alpha.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2B2B2B;
    color: #FFFFFF;
    font-family: Segoe UI, Arial;
    font-size: 11pt;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="commentLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnScreenshot">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Screenshot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblPath">
       <property name="text">
        <string>PATH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPath"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkPublish">
     <property name="text">
      <string>Publish to Server</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnOptions">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPublish">
       <property name="text">
        <string>Publish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resurces/ReferenceApp.qrc"/>
 </resources>
 <connections/>
</ui>
